src/Formatter/ClassNameFormatter.php

```diff
@@ -48,6 +48,6 @@ public function commandContext($command)
      */
     public function failureContext(array $currentContext, \Exception $e)
```

Comment:
I'm still wondering about this would it not be better the add the `command` as a argument here in case the commandContext returned empty? and maybe rename currentContext to commandContext to have it be clearer that we are "extending" the commandContext here.